### PR TITLE
Finalized translations

### PR DESCRIPTION
**Pull request by kairstenfay**
Translate the SCAN results report into Spanish, Vietnamese, and Chinese (simplified).

### CODE REVIEW

**Comment by kairstenfay** [comment]
[Update RoR Portal & RoR Results w/ PHSKC-approved translations](https://trello.com/c/iaS57pKI/529-update-ror-portal-ror-results-w-phskc-approved-translations)

**Review by joverlee521 — APPROVED** [review]
Looked great when I ran this locally 👍

File: scan/report-es.tex [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+  \link{https://www.doh.wa.gov/Portals/1/Documents/1600/coronavirus/COVIDexposed.pdf}{Patients
+  who were exposed to a confirmed COVID-19 case (Pacientes que estuvieron
+  expuestos a un caso confirmado de la COVID-19)}
+
+\item
+
+  \link{https://www.doh.wa.gov/Portals/1/Documents/1600/coronavirus/COVIDconcerned.pdf}{Unexposed
+  patients with COVID-19 symptoms (Pacientes que no estuvieron expuestos y que
+  tienen síntomas de la COVID-19)}
```
Comment (tsibley) [inline scan/report-es.tex]: WA DOH provides these documents in many other languages than English, so I think these links should point to the appropriate language, when available. For example, the Spanish version of the first link is https://www.doh.wa.gov/Portals/1/Documents/1600/coronavirus/COVIDcasepositiveSPANISH.pdf.

I don't know if that URL pattern is consistent, but I believe the matrix of documents × languages is listed on this page: https://www.doh.wa.gov/Emergencies/NovelCoronavirusOutbreak2020/HealthEducation

File: scan/report-es.tex [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+%\usepackage{showframe}
+
+\setlength{\headheight}{1.1in}
+\pagestyle{fancy}
+\fancyhf{}
+
+\lhead{\includegraphics[height=1in,clip]{logo}}
+\rhead{\pbox[b][1in][c]{\textwidth}{
+    \small
+    \textbf{Test Performed By}\\
```
Comment (tsibley) [inline scan/report-es.tex]: This phrase isn't translated in any of the versions, but it seems like it should be. An oversight when copying, or are we missing that phrase?

**Review by tsibley — CHANGES_REQUESTED** [review]

File: scan/report-es.tex (hunk 2 above)
Comment (kairstenfay) [reply scan/report-es.tex]: This phrase was missing from the translations. It is present in all of the translation documents as-is (see the [relevant Trello card](https://trello.com/c/iaS57pKI)).

File: scan/report-es.tex (hunk 1 above)
Comment (kairstenfay) [reply scan/report-es.tex]: I updated the translated documents for Spanish and Vietnamese. No translations were available for Chinese.

@tsibley Oddly enough, the latest commit that changes the URL of these links breaks the build. Do you have any idea why?

Comment (tsibley) [reply scan/report-es.tex]: > No translations were available for Chinese.

Wow, wasn't expecting that.

> Oddly enough, the latest commit that changes the URL of these links breaks the build. Do you have any idea why?

In LaTeX, underscores are special chars, so the ones in the new URLs need to be escaped like `\_`. :roll_eyes:

Comment (kairstenfay) [reply scan/report-es.tex]: Tricky LaTeX! It's now fixed. Thanks for the help.

File: scan/report-es.tex (hunk 2 above)
Comment (kairstenfay) [reply scan/report-es.tex]: "Test performed by" is now translated in all three documents.

**Review by tsibley — APPROVED** [review]

**Comment by kairstenfay** [comment]
Next steps:
- [x] Rebuild the image and push it up to Docker Hub (see README)
- [x] Update generate-pdfs in our backoffice automation to bump the image version it uses

**Pull request merged**